parts/k8s/containeraddons/ip-masq-agent.yaml

```diff
@@ -58,7 +58,11 @@ metadata:
 data:
   ip-masq-agent: |-
     nonMasqueradeCIDRs:
-      - <nonmasqCIDR>
-      - <nonmasqCNIIP>
-    masqLinkLocal: <masqLink>
+      - {{ContainerConfig "non-masquerade-cidr"}}
+    {{- if ContainerConfig "non-masq-cni-cidr"}}
```

Comment:
is the hyphen supposed to be here? `{{-`

Reply:
yes. It removes the preceding line break